From 07c4ac5300f5077e3b9bd925b07f02c6d51f5f7a Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Sun, 30 Jan 2022 17:35:33 +0100
Subject: [PATCH] kABI padding for qat
Patch-mainline: Never, kABI padding
References: bsn#129 bsc#1179531
Modified-by-SEL: Yes, modified due to different context

private structures of the qat driver

Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/crypto/qat/qat_common/adf_accel_devices.h |    1 +
 drivers/crypto/qat/qat_common/adf_common_drv.h    |    2 ++
 2 files changed, 3 insertions(+)

--- a/drivers/crypto/qat/qat_common/adf_accel_devices.h
+++ b/drivers/crypto/qat/qat_common/adf_accel_devices.h
@@ -142,6 +142,7 @@ struct adf_hw_device_data {
 	u8 num_logical_accel;
 	u8 num_engines;
 	u8 min_iov_compat_ver;
+	void *suse_kabi_padding;
 } __packed;
 
 /* CSR write macro */
--- a/drivers/crypto/qat/qat_common/adf_common_drv.h
+++ b/drivers/crypto/qat/qat_common/adf_common_drv.h
@@ -47,6 +47,8 @@ struct service_hndl {
 	unsigned long start_status[ADF_DEVS_ARRAY_SIZE];
 	char *name;
 	struct list_head list;
+
+	void *suse_kabi_padding;
 };
 
 static inline int get_current_node(void)
